docs/reference.md

```diff
@@ -299,7 +299,7 @@ You can perform a GET request on the collection endpoints to fetch a list of ent
 There are entity endpoints for each type of entity:
 
 - `https://$BASEURL/api/v1/namespaces/{namespace}`
-- `https://$BASEURL/api/v1/namespaces/{namespace}/actions/{actionName}`
+- `https://$BASEURL/api/v1/namespaces/{namespace}/actions/[{packageName}/]{actionName}`
```

Comment:
@rabbah Is this correct, to add [{packageName}/] this what you reported that need to be fix in docs.